code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -279,42 +282,43 @@
 				back_to_hunt()
 				return
 
-			if(iscarbon(target) && target.canBeHandcuffed())
-				if(!arrest_type)
-					if(!target.handcuffed)  //he's not cuffed? Try to cuff him!
-						start_cuffing(target)
-					else
-						back_to_idle()
-						return
-			else
+			if(arrest_type) // should we not cuff?
```

Comment:
if arrest_type is TRUE, shouldn't we be handcuffing, not going back to idle?

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/61080616/d1c90029-d3b0-4501-a85f-d343f682f237)

Reply:
changed var name to `no_handcuffs`, i hope this will be better for understanding, will need to update TGUI window to change data variable